pkg/operator/operator.go

```diff
@@ -84,13 +78,35 @@ func NewOpenShiftControllerManagerOperator(
 	return c
 }
 
+// getOperatorConfig execute get request using operator config client.
+func (c OpenShiftControllerManagerOperator) getOperatorConfig() (
+	*operatorapiv1.OpenShiftControllerManager, error,
+) {
+	return c.operatorConfigClient.OpenShiftControllerManagers().
```

Comment:
since we are already have an operatorConfigInformer set up, we should be using a lister off of that to fetch the config and avoid hits against the api server

a few releases ago Clayton dinged a bunch of operators for for not minimizing there hits against the api server for stuff like this

Reply:
bump on this one @otaviof .... is this on your to-do list?  Or is it in progress but just not pushed yet?

Reply:
Please consider latest commit.

Reply:
@otaviof looks like this needs to be updated to use a lister cache.

Reply:
@adambkaplan it should be present on commit c7a5f0af, am I overlooking something?